bots/image-create

```diff
@@ -147,7 +147,7 @@ class MachineBuilder:
         if args.no_build:
             source = None
         else:
-            source = subprocess.check_output([ os.path.join(BOTS, "make-source") ]).strip()
+            source = subprocess.check_output([ os.path.join(BOTS, "make-source") ]).decode('utf-8').strip()
```

Comment:
FTR, I'm fine with this. But just for the educational aspect and for other projects: This usually causes trouble with Python 2. You can make subprocess calls bilingual with `universal_newlines=True` which will DTRT with both Python 2 and 3.

Reply:
Fixed.